programs/drift/src/controller/orders.rs

```diff
@@ -1227,38 +1224,22 @@ fn fulfill_perp_order(
     valid_oracle_price: Option<i64>,
     now: i64,
     slot: u64,
-    market_is_reduce_only: bool,
     amm_is_available: bool,
 ) -> DriftResult<(u64, bool, bool)> {
     let market_index = user.orders[user_order_index].market_index;
 
-    let position_index = get_position_index(&user.perp_positions, market_index)?;
-    let order_direction = user.orders[user_order_index].direction;
-    let position_base_asset_amount_before = user.perp_positions[position_index].base_asset_amount;
-    let risk_decreasing = is_order_risk_decreasing(
-        &order_direction,
-        user.orders[user_order_index]
-            .get_base_asset_amount_unfilled(Some(position_base_asset_amount_before))?,
-        position_base_asset_amount_before.cast()?,
-    )?;
-
-    if !risk_decreasing && market_is_reduce_only {
-        cancel_risk_increasing_order(
```

Comment:
when market if reduce only, we force all orders to be reduce_only. and we already check to cancel reduce only orders elsewhere, so this was redundant now

Reply:
so even if a risk increasing limit order was placed prior to the market becoming reduce only, it will cancel still?

Reply:
no it would still be able to increase the position

Reply:
we can add this back else where